src/test/java/de/kherud/llama/LlamaModelIT.java

```diff
@@ -157,14 +171,13 @@ public void testCompleteGrammar() {
 	@Test
 	public void testEmbedding() {
 		float[] embedding = model.embed(prefix);
-		Assert.assertEquals(5120, embedding.length);
+		Assert.assertEquals(4096, embedding.length);
 	}
 
 	@Test
 	public void testTokenization() {
 		String prompt = "Hello, world!";
 		int[] encoded = model.encode(prompt);
-		Assert.assertArrayEquals(new int[]{15043, 29892, 3186, 29991}, encoded);
```

Comment:
The token IDs are different because the model is different.  I figured that a sufficient test is that the encode/decode can be round-tripped.